libparsec/crates/protocol/schema/authenticated_cmds/shamir_create_shared_recovery_device.json5

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "shamir_recovery_setup",
```

Comment:
File name (`shamir_create_shared_recovery_device`) should match the command name